.github/workflows/ci.yml

```diff
@@ -1,8 +1,9 @@
-name:
+name: Meshery Adapter Library
+
 on:
   push:
     branches:
-      - 'master'
+      - '**'
```

Comment:
Is this mean of all the PR would run the CI?

Reply:
It means that the CI runs on any branch one pushes to, especially also any branch in personal forks.

Reply:
If I remember right, Github decreased the number of CI of the repo, can these make 0 effect to us?

Reply:
I haven't thought about that (but heard about it too). As most branches exist in personal forks, I think it would not contribute to the Layer5 budget. Let's keep an eye on it. Would be interested to see whether there is a count somewhere that shows usage...

Reply:
https://github.com/pricing

> 2,000 Actions minutes/month 
> Free for public repositories 

https://docs.github.com/en/free-pro-team@latest/actions/reference/usage-limits-billing-and-administration#about-billing-for-github-actions

> GitHub Actions usage is free for public repositories and self-hosted runners.

https://docs.github.com/en/free-pro-team@latest/actions/reference/usage-limits-billing-and-administration#usage-limits